modules/postgres/postgres_test.go

```diff
@@ -171,6 +209,54 @@ func TestWithConfigFile(t *testing.T) {
 	defer db.Close()
 }
 
+func TestWithSSL(t *testing.T) {
+	ctx := context.Background()
+
+	caCert, serverCerts, err := createSSLCerts(t)
+	require.NoError(t, err)
+
+	container, err := postgres.RunContainer(ctx,
```

Comment:
bug: let's use non-deprecated API
```suggestion
	container, err := postgres.Run(ctx,
```